lib/buckaroo.ex

```diff
@@ -15,7 +15,12 @@ defmodule Buckaroo do
     socket = if s = opts[:socket], do: {s, opts[:socket_opts] || opts[:opts] || []}
 
     options =
-      [port: 3000, dispatch: [{:_, [{:_, __MODULE__, {socket || router, router}}]}]]
+      [
+        port: 3000,
+        compress: true,
+        protocol_options: [idle_timeout: :infinity],
```

Comment:
Don't know if `:infinity` should be default, which is needed for proper `SSE` or just document it properly.

Reply:
Resolved by removing the default and adding an extended check that will warn if `SSE` is used, but the timeout is not `inifinity`.